src/Docker.DotNet/Endpoints/ContainerOperations.cs

```diff
@@ -112,6 +112,25 @@ public Task GetContainerLogsAsync(string id, ContainerLogsParameters parameters,
                 progress);
         }
 
+        public async Task<MultiplexedStream> GetContainerLogsAsync(string id, bool tty, ContainerLogsParameters parameters, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (string.IsNullOrEmpty(id))
+            {
+                throw new ArgumentNullException(nameof(id));
+            }
+
+            if (parameters == null)
+            {
+                throw new ArgumentNullException(nameof(parameters));
+            }
+
+            IQueryString queryParameters = new QueryString<ContainerLogsParameters>(parameters);
+
+            Stream result = await this._client.MakeRequestForStreamAsync(new[] { NoSuchContainerHandler }, HttpMethod.Get, $"containers/{id}/logs", queryParameters, cancellationToken).ConfigureAwait(false);
```

Comment:
The `MakeRequestForHijackedStreamAsync` code does't work? It returns the `WriteClosableStream` already and then we dont need the changes in the `MultiplexedStream`. Also, should we keep the `IProgress` model and just use `IProgress<string> stdout, IProgress<string> stderr` instead so its still fully async?